sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/google/GoogleHttpClient.java

```diff
@@ -62,35 +62,35 @@
     private final String urlDomain;
     private Authentication lastUsedAuthentication;
 
-    public GoogleHttpClient(final String urlDomain) {
-        this.urlDomain = urlDomain;
+    /**
+     * Request initializer called on every request made by the requestFactory
+     */
+    private HttpRequestInitializer requestInitializer = request -> {
+        request.setResponseInterceptor(response -> {
+            // retrieve new LWSSO in response if any
+            HttpHeaders responseHeaders = response.getHeaders();
+            updateLWSSOCookieValue(responseHeaders);
+        });
 
-        //String clientTypeHeader = authentication.getClientHeader();
+        request.setUnsuccessfulResponseHandler((httpRequest, httpResponse, b) -> false);
 
-        HttpTransport HTTP_TRANSPORT = new NetHttpTransport();
-        requestFactory = HTTP_TRANSPORT
-                .createRequestFactory(request -> {
-                    request.setResponseInterceptor(response -> {
-                        // retrieve new LWSSO in response if any
-                        HttpHeaders responseHeaders = response.getHeaders();
-                        updateLWSSOCookieValue(responseHeaders);
-                    });
-                    request.setUnsuccessfulResponseHandler((httpRequest, httpResponse, b) -> false);
-
-                    if (lwssoValue != null && !lwssoValue.isEmpty()) {
-                        request.getHeaders().setCookie(LWSSO_COOKIE_KEY + "=" + lwssoValue);
-                    }
+        if (lwssoValue != null && !lwssoValue.isEmpty()) {
+            request.getHeaders().setCookie(LWSSO_COOKIE_KEY + "=" + lwssoValue);
+        }
 
-                    if(lastUsedAuthentication != null) {
-                        String clientTypeHeader = lastUsedAuthentication.getClientHeader();
-                        if (clientTypeHeader != null && !clientTypeHeader.isEmpty()) {
-                            request.getHeaders().set(HPE_CLIENT_TYPE, clientTypeHeader);
-                        }
-                    }
-                    request.setReadTimeout(60000);
-                });
+        if(lastUsedAuthentication != null) {
+            String clientTypeHeader = lastUsedAuthentication.getClientHeader();
+            if (clientTypeHeader != null && !clientTypeHeader.isEmpty()) {
+                request.getHeaders().set(HPE_CLIENT_TYPE, clientTypeHeader);
+            }
+        }
+        request.setReadTimeout(60000);
+    };
 
-        //authenticate();
+    public GoogleHttpClient(final String urlDomain) {
```

Comment:
Can you start the class with the constructor.  The private method above should be below